tensorflow/compiler/tf2tensorrt/kernels/trt_engine_op.cc

```diff
@@ -306,7 +311,23 @@ TRTEngineOp::TRTEngineOp(OpKernelConstruction* context)
     use_implicit_batch_ = true;
   }
 #endif
-  if (!use_implicit_batch_) {
+  if (use_implicit_batch_) {
+    if (input_partial_shapes_.size() == 0) {
+      VLOG(1) << "Attribute input_shapes it not set. This happens probably "
+              << "because you are using a model that is already converted "
+              << "to TensorRT (i.e. includes TRTEngineOp in graph). If you "
+              << "convert the original model again to TensorRT, the "
+              << "attributes input_shapes will be set automatically.";
```

Comment:
I don't quite understand this error message after its first sentence. Do you mean:
You are likely trying to convert a model that is already converted to TensorRT.  You should convert the original model instead.